test/studies/prk/stencil/PERFCOMPOPTS

```diff
@@ -0,0 +1 @@
+-sassertNoSlicing
```

Comment:
Personally, I don't think we should be throwing `-sassertNoSlicing` in any performance tests. 

`-sassertNoSlicing` is currently being tested in one of the playgrounds, but I don't think it's a good idea to throw that big hammer in real tests. @bradcray might disagree though, and if he thinks it's ok I won't object.